types/wlr_output_management_v1.c

```diff
@@ -761,6 +761,16 @@ static void manager_send_head(struct wlr_output_manager_v1 *manager,
 			output->phys_width, output->phys_height);
 	}
 
+	if (version >= ZWLR_OUTPUT_HEAD_V1_MAKE_SINCE_VERSION) {
```

Comment:
Can we not send anything if these are empty strings?

Reply:
I tried with strnlen and strnlen_s but they are not declared. What's the right way to check if the string is empty in C (could be not null-terminated)?

Reply:
Checking the first element of the strings is possible though because initialized to 0 and then `make[0] == '\0' == 0`. Patch updated.